src/vs/base/worker/workerMain.ts

```diff
@@ -103,11 +103,18 @@
 	}
 
 	function configureAMDLoader() {
+		const nlsConfig: any = {};
+		nlsConfig['vs/nls'] = {
+			availableLanguages: {
+				'*': navigator.language.toLowerCase()
```

Comment:
I think we want to use a similar change as here:
https://github.com/microsoft/vscode/blob/cf1b0ce3d660a35a7060dbac9be7c045293ebaee/src/vs/code/browser/workbench/workbench.html#L46-L56

where we look at local storage as well so that if the user has configured a display language different than their browser that still works.

Reply:
One thing I'm curious about is if this works in Desktop. I would guess that electron does all the hard work of plumbing the language to `navigator.language` but it would be good to confirm.

Reply:
Yes, I have also observed the need to read the localstorge before. However, there is a problem. Similar to `vscode.dev`, the domain name is inconsistent with the domain where the iframe is located. Or we need to let the main process tell the iframe after it gets the localstorge, and then tell the worker. 

At the same time, I will try to see the electron treatment.